tests/Agent/IntegrationTests/UnboundedIntegrationTests/MsSql/MsSqlQueryParamTests.cs

```diff
@@ -205,34 +205,6 @@ public MsSqlQueryParamTests_SystemData_NoAtSigns_FWLatest(ConsoleDynamicMethodFi
 
     #region System.Data.SqlClient (.NET Core/5+ only)
 
-    // Set to net7 since the package this relies on does not currently support net8.
```

Comment:
We should try to figure out why we can test ms sql on net8 so that we can keep these tests.

Reply:
It's all coming back to me now... System.Data.SqlClient isn't usable in .NET 8... https://github.com/dotnet/runtime/issues/86969